test/test_utilities.cpp

```diff
@@ -93,6 +93,27 @@ void SetNGraphTFBackend(const string& backend_name) {
   SetEnvVariable("NGRAPH_TF_BACKEND", backend_name);
 }
 
+// Generating Random Seed
+int GetSeedForRandomFunctions() {
+  string env_name = "NGRAPH_TF_SEED";
+  int seed = static_cast<unsigned>(time(0));
+  if (!IsEnvVariableSet(env_name)) {
+    NGRAPH_VLOG(5) << "Got seed " << seed;
+    return seed;
+  }
+
+  string seedstr = GetEnvVariable(env_name);
+  try {
+    seed = stoi(seedstr);
+  } catch (const std::exception& exp) {
+    throw std::runtime_error{"Cannot set " + env_name + " with value " +
+                             seedstr + " ,got exception " + exp.what()};
```

Comment:
So this allows you to set seeds externally through `NGRAPH_TF_SEED`. But in CI, will we be setting `NGRAPH_TF_SEED`? In which case CI will be using time as seed. If we expect the test to fail for certain seeds, we should dump the seed value on failure so that they can be repro-ed later